2d/camera-zoom/Actors/ZoomingCamera2D.gd

```diff
@@ -11,6 +13,10 @@ var _zoom_level := 1.0 setget _set_zoom_level
 onready var tween: Tween = $Tween
 
 
+func _ready() -> void:
+	EVENTS.connect("zoom_slider_changed", self, "_on_zoom_slider_changed")
```

Comment:
Avoid using a singleton in these demos. It's a decoupling pattern for more complex games or code.

Reply:
Also, it's not a constant, but a node, so the name should be `Events`.